admin/database/postgres/migrations/0007.sql

```diff
@@ -0,0 +1,102 @@
+VALUES
+    ('12345678-0000-0000-0000-000000000001', 'admin', true, true, true, true, true, true, true),
+    ('12345678-0000-0000-0000-000000000002', 'collaborator', true, false, true, false, false, true, false),
+    ('12345678-0000-0000-0000-000000000003', 'reader', true, false, true, false, false, true, false);
+
+INSERT INTO project_roles (id, name, read_project, manage_project, read_prod_branch, manage_prod_branch, read_dev_branches, manage_dev_branches, read_project_members, manage_project_members)
+VALUES
+    ('12345678-0000-0000-0000-000000000001', 'admin', true, true, true, true, true, true, true, true),
+    ('12345678-0000-0000-0000-000000000002', 'collaborator', true, false, true, false, true, true, true, false),
+    ('12345678-0000-0000-0000-000000000003', 'reader', true, false, true, false, true, false, false, false);
+
+CREATE TABLE users_orgs_roles (
+    user_id UUID NOT NULL REFERENCES users (id) ON DELETE CASCADE,
+    org_id UUID NOT NULL REFERENCES organizations (id) ON DELETE CASCADE,
+    org_role_id UUID NOT NULL REFERENCES org_roles (id) ON DELETE CASCADE,
+    PRIMARY KEY (user_id, org_id)
+);
+
+CREATE TABLE users_projects_roles (
+    user_id UUID NOT NULL REFERENCES users (id) ON DELETE CASCADE,
+    project_id UUID NOT NULL REFERENCES projects (id) ON DELETE CASCADE,
+    project_role_id UUID NOT NULL REFERENCES project_roles (id) ON DELETE CASCADE,
+    PRIMARY KEY (user_id, project_id)
+);
+
+-- user group related table operations
+CREATE TABLE usergroups (
+    id UUID DEFAULT uuid_generate_v4() PRIMARY KEY,
+    org_id UUID NOT NULL REFERENCES organizations (id) ON DELETE CASCADE,
+    display_name TEXT NOT NULL,
+    description TEXT
+);
+
+-- create all users group for existing organizations
+INSERT INTO usergroups (org_id, display_name) SELECT id, 'all_users' FROM organizations;
+
+-- add all_group_id column to the organizations table referencing usergroups table
+ALTER TABLE organizations ADD COLUMN all_group_id UUID REFERENCES usergroups (id) ON DELETE RESTRICT;
+
+-- insert all_group_id into the organizations table for existing organizations
+UPDATE organizations SET all_group_id = (SELECT id FROM usergroups WHERE org_id = organizations.id AND display_name = 'all_users');
+
+-- ALTER TABLE organizations ALTER COLUMN all_group_id SET NOT NULL;
+
+CREATE TABLE users_usergroups (
+    user_id UUID NOT NULL REFERENCES users (id) ON DELETE CASCADE,
+    group_id UUID NOT NULL REFERENCES usergroups (id) ON DELETE CASCADE
+);
+
+-- insert all users into all_users_group of corresponding organizations
+WITH user_org AS (
+    SELECT users.id as user_id, users_orgs_roles.org_id as org_id FROM users JOIN users_orgs_roles ON users_orgs_roles.user_id = users.id
+)
+INSERT INTO users_usergroups (user_id, group_id)
+SELECT user_org.user_id, org.all_group_id FROM user_org JOIN organizations org ON org.id = user_org.org_id;
```

Comment:
Since `users_orgs_roles` was created in this migration, isn't this a no-op?

Reply:
hmm yeah thats correct, how should we assign users to org then. For existing users, what roles should be added and who should become the org and project admins.

Reply:
As discussed, assuming clean slate here so removing all assignments.